mne_features/univariate.py

```diff
@@ -1013,6 +1016,76 @@ def compute_svd_entropy(data, tau=2, emb=10):
     return -np.sum(np.multiply(sv_norm, np.log2(sv_norm)), axis=-1)
 
 
+def compute_powercurve_deviation(sfreq, data, fmin=0.1, fmax=50, 
+                                 with_intercept=True):
+    """ Linear fit to the log-log frequency-curve with fit error (per channel).
+
+    Parameters
+    ----------
+    data : ndarray, shape (n_channels, n_times)
+
+    freq_range: list, length 2
+        Gives floats fmin, fmax that define frequency range of the linear fit.
+
+    sfreq: float
+        The sampling frequency.
+
+    Returns
+    -------
+    output : ndarray, shape (n_channels * 4,)
+        The 4 characteristics are intercept, slope, MSE, and R2 per channel.
+
+    Notes
+    -----
+    Alias of the feature function: **powercurve_deviation**. See [1]_
+    and [2]_.
+
+    References
+    ----------
+    .. [1] Demanuele C, James CJ, Sonuga-Barke EJ.
```

Comment:
Travis is not green because of a non-ASCII character somewhere in those two refs. This is probably due to a copy-paste... If you're using Pycharm, the editor will tell you where the character is.